bears/go/GoTOMLBear.py

```diff
@@ -0,0 +1,22 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.GoRequirement import GoRequirement
+
+
+@linter(executable='gotoml',
+        use_stdin=True,
+        output_format='corrected')
+class GoTOMLBear:
+    """
+    It is used to the know the type of every key used in a TOML file.
```

Comment:
This is not the purpose of the linter, see https://github.com/BurntSushi/toml/tree/master/cmd/tomlv or https://github.com/BurntSushi/toml/blob/master/cmd/tomlv/main.go#L1